shot-android/build.gradle

```diff
@@ -4,14 +4,15 @@ apply plugin: "org.jlleitschuh.gradle.ktlint"
 apply from: '../release.gradle'
 
 android {
-    compileSdkVersion 29
```

Comment:
Why decreasing the compile and target SDK Version from 29 to 28?

Reply:
Robolectirc requests JDK9 to use compileSdkVersion 29. However, if we use SDK version 28 we can keep using JDK8 which is the one we need for several projects 😃